docs/source/gbd2019_models/risk_exposures/wasting_state_exposure/index.rst

```diff
@@ -589,9 +589,50 @@ for now, this unknown is :math:`dur\_cat3`, which we will assume to be :math:`1/
 find values from the literature with which to update this. Note that the below
 equations also contain unknowns for :math:`d_1, d_2, d_3, d_4, f_1, f_2, f_3, f_4, p_0`.
 These are calculable from GBD data, but as these vary per age/sex category, we 
-leave them as variables here.
+leave them as variables here. Below we include equations for how these are calculated, and a link to a notebook implementing these equations.
 
-.. todo:: fill in the :math:`f_i` and :math:`d_i` vars with values/eqns
+.. math::
+
+   d_{cat 1&2}: acmr + sum([(emr_i*prev_i - cmsr_i) for i in diar,lri,msl]) + (emr_pem*1 - cmsr_pem)
```

Comment:
Doesn't this imply that the only difference in mortality between cats1&2 and cats3&4 is that cats1&2 have PEM? Shouldn't there be some reference to the RRs for each of these diseases?

Reply:
The RRs of the disease determine the prevalence of the disease in the category. The CSMRs of the diseases are the same no matter what category you are in. prev_i is category specific.